src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -307,6 +307,7 @@ internal void SendInvocationRequest(ScriptInvocationContext context)
                     {
                         FunctionId = functionMetadata.FunctionId,
                         InvocationId = context.ExecutionContext.InvocationId.ToString(),
+                        CorrelationId = Activity.Current.Id,
```

Comment:
```suggestion
                        Traceparent = Activity.Current?.Id,
```

There is a long thread about 'Traceparent' vs `TraceParent` in W3C community and consensus is `Traceparent` or `traceparent`

. I prefer `Traceparent` as well, but leave it up to you to decide.

Reply:
Also as we discussed offline there will be `Tracestate` coming very soon - could you add it here and in proto files/workers now so we won't need to update them again?

Reply:
cc @AnatoliB  - as he asked similar question on casing on the powershell worker changes
@yojagad / @lmolkova - does it effect functionally Traceparent vs TraceParent?

Reply:
@pragnagopa Functionally, things won't be different is my understanding whether we choose to use "Traceparent" or "TraceParent". It's more about sticking to W3C community standards by choosing "Traceparent". Please correct me if I'm wrong @lmolkova